src/tsVariableTypes/task2.ts

```diff
@@ -0,0 +1,5 @@
+const stringsArray: string[] = ["H", "Ha", "Hay"];
+const numArray: number[] = [];
+
+stringsArray.map((word: string) => numArray.push(word.length));
```

Comment:
`(word: string)` sioje vietoje nebereikia nurodyti tipo, jis jau aiskus is pirmos Eilgutes